### DISCUSSION TITLE
Install to RHEL 8

### DISCUSSION THREAD

**Discussion by troyd1000** [ImageMagick/ImageMagick#3388] [General]
I search on how to install in rhel 8 and it said to install graphicsmagick instead.  I did, but there was no convert.  I then had someone tell me to go to the IM download page and install the redhat 8 version.  I downloaded ImageMagick-7.0.11-3.x86_64.rpm and ImageMagick-libs-7.0.11-3.x86_64.rpm and installed as instructed.  I got the lot of dependency errors.  How do i clear this up?  Did I forget to install something else?

Here is the output:
sh-4.4$ sudo rpm -Uvh ImageMagick-7.0.11-3.x86_64.rpm
error: Failed dependencies:
        ImageMagick-libs(x86-64) = 7.0.11-3 is needed by ImageMagick-7.0.11-3.x86_64
        libMagickCore-7.Q16HDRI.so.9()(64bit) is needed by ImageMagick-7.0.11-3.x86_64
        libMagickWand-7.Q16HDRI.so.9()(64bit) is needed by ImageMagick-7.0.11-3.x86_64
        libMagickWand-7.Q16HDRI.so.9(VERS_9.0)(64bit) is needed by ImageMagick-7.0.11-3.x86_64
        libfftw3.so.3()(64bit) is needed by ImageMagick-7.0.11-3.x86_64
        libraqm.so.0()(64bit) is needed by ImageMagick-7.0.11-3.x86_64
sh-4.4$ sudo rpm -Uvh ImageMagick-libs-7.0.11-3.x86_64.rpm
error: Failed dependencies:
        libHalf.so.12()(64bit) is needed by ImageMagick-libs-7.0.11-3.x86_64
        libIex-2_2.so.12()(64bit) is needed by ImageMagick-libs-7.0.11-3.x86_64
        libIexMath-2_2.so.12()(64bit) is needed by ImageMagick-libs-7.0.11-3.x86_64
        libIlmImf-2_2.so.22()(64bit) is needed by ImageMagick-libs-7.0.11-3.x86_64
        libIlmThread-2_2.so.12()(64bit) is needed by ImageMagick-libs-7.0.11-3.x86_64
        libImath-2_2.so.12()(64bit) is needed by ImageMagick-libs-7.0.11-3.x86_64
        libfftw3.so.3()(64bit) is needed by ImageMagick-libs-7.0.11-3.x86_64
        libraqm.so.0()(64bit) is needed by ImageMagick-libs-7.0.11-3.x86_64
        libraw_r.so.19()(64bit) is needed by ImageMagick-libs-7.0.11-3.x86_64
        libwmflite-0.2.so.7()(64bit) is needed by ImageMagick-libs-7.0.11-3.x86_64
sh-4.4$

**Comment by urban-warrior** [comment]
https://www.itzgeek.com/post/how-to-install-imagemagick-and-php-imagick-on-centos-8-rhel-8/
